src/views/wiki/immunizationWiki.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  Box,
+  Divider,
+  Flex,
+  Icon,
+  Text,
+  useBoolean,
+  useColorModeValue,
+} from '@chakra-ui/react';
+import {
+  getSomeVaccinations,
+  Vaccination,
+} from '../../core/mockData/vaccination';
+import { ChevronRightIcon, InfoIcon } from '@chakra-ui/icons';
+import { Link } from 'react-router-dom';
+import { MockRecommendations } from '../../core/mockData/mockRecommendation';
+import { getColorByStatus, getIconByStatus } from '../../theme/theme';
+import React from 'react';
+
+export function ImmunizationWiki() {
+  const vaccinations: Vaccination[] = getSomeVaccinations();
+  const [showInfo, setShowInfo] = useBoolean();
+  // Into each entry of a disease, add whether the user has a recommendation getting an immunization against that disease
+  // TODO: Include completed immunizations
+  vaccinations.map((disease) => {
```

Comment:
Shouldn't this be diseases instead of vaccinations? Because your listing diseases in the wiki.